services/meta/data.go

```diff
@@ -37,6 +37,8 @@ type Data struct {
 
 	MaxShardGroupID uint64
 	MaxShardID      uint64
+
+	DefaultRetentionPolicyName string `json:"-"`
```

Comment:
It ensures it doesn't get serialized with the JSON file in the open source version.